packages/paste-core/components/tabs/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@twilio-paste/tabs",
+  "version": "0.0.0",
+  "category": "interaction",
+  "status": "alpha",
+  "description": "Twilio's Tabs component.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && yarn compile",
+    "build:dev": "yarn clean && yarn compile:dev",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist && rm -rf tsconfig.build.tsbuildinfo && rm -rf .rpt2_cache",
+    "compile": "rollup -c --environment NODE_ENV:production",
+    "compile:dev": "rollup -c --environment NODE_ENV:development",
+    "prepublishOnly": "yarn build",
+    "type-check": "tsc --noEmit"
+  },
+  "peerDependencies": {
+    "@emotion/core": "^10.0.10",
+    "@emotion/styled": "^10.0.10",
+    "@twilio-paste/box": "^2.4.8",
+    "@twilio-paste/flex": "^0.3.27",
+    "@twilio-paste/style-props": "1.1.9",
+    "@twilio-paste/tabs-primitive": "^0.1.0",
+    "@twilio-paste/text": "^2.1.9",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "styled-system": "^5.1.2"
```

Comment:
Most recently for peer dependencies, after fixing a bunch of @yarnpkg/doctor warnings (now seems to be working again), I came across this article explaining the `Unmet transitive peer dependency` warning. https://dev.to/arcanis/implicit-transitive-peer-dependencies-ed0. The crux of it was, peer deps of peer deps should also be listed.

Otherwise you get a whole bunch of these looking errors
![image](https://user-images.githubusercontent.com/368249/83917372-95fc6580-a72b-11ea-8193-997c88878ec9.png).

In some packages that causes the compiled code to blow out with some peer deps that it needs that aren't listed. In other cases like this one, they seem to compile fine without it.

I'm been leaning on the side of caution and just listing them as it does no harm and clears the warnings about unmet peers

Reply:
:O yarn doctor works again?! 
Will do! 👍